src/getrepos.ts

```diff
@@ -9,7 +9,7 @@ export async function getRepoIds(repoPath: string, clientInput: {account, access
   const j1Client = await getClient(clientInput);
 
   for (const repo of repos) {
-    if (repo === '.DS_Store') {
+    if (repo.startsWith('.')) {
```

Comment:
Could this be an lstat directory check as well? Do you ever want to query J1 for file names, regardless of hidden or not?